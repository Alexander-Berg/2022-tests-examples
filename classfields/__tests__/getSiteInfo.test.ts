import { getSiteInfo } from '../getSiteInfo';

describe('getSiteInfo', () => {
    it('Возвращает корректый объект', () => {
        expect(getSiteInfo({ siteId: 1 })).toMatchSnapshot();
    });
});
