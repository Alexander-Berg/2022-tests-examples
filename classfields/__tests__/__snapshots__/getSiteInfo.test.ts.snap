// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getSiteInfo Возвращает корректый объект 1`] = `
Object {
  "siteId": "1",
}
`;
